python/tk_desktop/about_screen.py

```diff
@@ -20,11 +22,16 @@ class AboutScreen(QtGui.QDialog):
     def __init__(self, body="", parent=None):
         QtGui.QDialog.__init__(self, parent)
 
+        year = date.today().year
+
         # setup the GUI
         self.ui = about_screen.Ui_AboutScreen()
         self.ui.setupUi(self)
         self.ui.body.setText(body)
         self.ui.licensesButton.clicked.connect(self._show_licenses)
+        self.ui.copyright.setText(
+            f"Copyright ©{year} ShotGrid Software Inc.<br/>All rights reserved."
```

Comment:
The dynamic year cannot be set during execution. It must be set during build/compilation/release IMO

Reply:
I see your point and it makes sense. However, I can't see where I can make a dynamic year without affecting the whole packing process. Let's bump it to 2023 for now, to avoid extending this ticket too much.